unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Main/Main.cs

```diff
@@ -106,33 +106,8 @@ protected virtual void SetupServices()
             Environment.Setup(ServiceLocatorFactory.CreateDefault());
         }
 
-        protected virtual async UniTaskVoid Start()
+        protected virtual void Start()
         {
-            var rpc = Environment.i.serviceLocator.Get<IRPC>();
-
-            await rpc.EnsureRpc();
-            
-            await UniTask.SwitchToMainThread();
-
-            // this event should be the last one to be executed after initialization
-            // it is used by the kernel to signal "EngineReady" or something like that
-            // to prevent race conditions like "SceneController is not an object",
-            // aka sending events before unity is ready
-            ClientAnalyticsKernelService analytics = rpc.Analytics();
-
-            await analytics.SystemInfoReport(new SystemInfoReportRequest()
-            {
-                GraphicsDeviceName = SystemInfo.graphicsDeviceName,
-                GraphicsDeviceVersion = SystemInfo.graphicsDeviceVersion,
-                GraphicsMemorySize = (uint)SystemInfo.graphicsMemorySize,
-                ProcessorType = SystemInfo.processorType,
-                ProcessorCount = (uint)SystemInfo.processorCount,
-                SystemMemorySize = (uint)SystemInfo.systemMemorySize,
-
-                // TODO: remove useBinaryTransform after ECS7 is fully in prod
-                UseBinaryTransform = true,
-            });
-
             // We trigger the Decentraland logic once everything is initialized.
             WebInterface.StartDecentraland();
```

Comment:
This function, starts the engine calling the web directly. And from the Kernel side, starts everything, including the RPC.

So I had to move the rpc call after this `StartDecentraland()`